Client/Frontend/Strings.swift

```diff
@@ -42,6 +42,14 @@ extension Strings {
     public static let ASRemoveButton = NSLocalizedString("AS.Delete.Button", value: "Delete", comment: "Button shown in editing mode to remove this site from the top sites panel.")
     public static let ASCancelButton = NSLocalizedString("AS.Cancel.Button", value: "Cancel", comment: "Button shown to cancel editing of top site.")
     public static let ASHighlightsTitle =  NSLocalizedString("AS.Highlights.SectionTitle", value: "Highlights", comment: "Section title label for Highlights")
+
```

Comment:
Although, (This is my fault, sorry) some of the strings have "AS.(blah)" while others are "ActivityStream.(blah)". Lets rewrite this so they are all ActivityStream.(blah)"